src/applications/check-in/pre-check-in/pages/Introduction/IntroductionDisplay.jsx

```diff
@@ -73,16 +87,15 @@ const IntroductionDisplay = props => {
       if (e?.key && e.key !== ' ') {
         return;
       }
+      let slug = `pre-check-in-started-${isPhone ? 'phone' : 'in-person'}`;
+      if (isPreCheckInActionLinkTopPlacementEnabled) slug += '-top-position';
```

Comment:
Should we have `-bottom-position` added in the inverse? The viewed intro event will have both.

Reply:
good question, @benbrasso-agile6 and I agreed not to so that the existing event name won't change and affect historical stats.